src/TaskItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const TaskItem = ({ task, toggleCompleted }) => {
+    const {id, name, due, completed} = task;
+
+    const toggleCompletedClick = () => {
+        toggleCompleted(id);
+    }
+    return (
+        <tr className={(new Date(due)) <= (new Date()) ? 'overdue' : 'notDueYet'}>
+            <td><input type='checkbox' checked={completed} onChange={toggleCompletedClick} /></td>
```

Comment:
Personal choice but making such tiny function a standalone variable seems redundant to me.
```suggestion
            <td><input type='checkbox' checked={completed} onChange={() => toggleCompleted(id)} /></td>
```